scrapers/AbbyWinters.yml

```diff
@@ -0,0 +1,28 @@
+name: Abby Winters
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.abbywinters.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+      Performers:
+        Name:
+          selector: //td/a[starts-with(@href,'https://www.abbywinters.com/nude_girl/')]/text()
+      Title:
+        selector: //div[starts-with(@class,'page-heading')]/div/h1/text()
+      Details:
+        selector: //div[@class='description padd-bottom-15 padd-left-15 padd-right-15']/p/span/text()
```

Comment:
This selector wasn't working on some scene pages like `https://www.abbywinters.com/girls_and_their_boys/thumb_in_bum/anjali_nadir`.

This seems to work:
```
selector: //div[@class[contains(.,'description')]]//text()
concat: " "
```